include/akushon/config/grpc/config.hpp

```diff
@@ -42,7 +42,8 @@ class ConfigGrpc
 
   ~ConfigGrpc();
 
-  void Run(uint16_t port, const std::string & path, rclcpp::Node::SharedPtr & node,
+  void Run(
+    const std::string & path, rclcpp::Node::SharedPtr & node,
```

Comment:
`node` should be passed as a constant reference:
```suggestion
    const std::string & path, const rclcpp::Node::SharedPtr & node,
```